package-lock.json

```diff
@@ -26,124 +26,250 @@
       }
```

Comment:
Are you 100% sure this file should be in the repository? To me it looks like some kind of automatically generated file, which is the same as expelling it from the repo :P